pymc3/distributions/transforms.py

```diff
@@ -116,13 +119,16 @@ def __init__(self, a, b, eps=1e-6):
 
     def backward(self, x):
         a, b = self.a, self.b
-        r = (b - a) / (1 + tt.exp(-x)) + a
+        r = (b - a) * tt.nnet.sigmoid(x) + a
         return r
 
     def forward(self, x):
         a, b, e = self.a, self.b, self.eps
-        r = tt.log(tt.maximum((x - a) / tt.maximum(b - x, e), e))
-        return r
+        return tt.log(x - a) - tt.log(b - x)
+
+    def jacobian_det(self, x):
```

Comment:
Your right, the problem is, where should we put something like that? Putting some long latex stuff in the docstring is not exactly readable:

# Interval transform


Forward:
$$
\phi: [a, b] \to \textbf{R}\\
x \mapsto \text{logit}(\tfrac{x - a}{b - a}) = \log(x - a) - \log(b - x)
$$

Backward:
$$
\phi^{-1}: y\mapsto a + (b − a) \cdot \text{sigmoid}(y)
$$

Jacobian:
$$
\left| \frac{d}{dy} \phi^{-1} \right|
= \frac{1}{1 + e^{-x}} \cdot \left(1 - \frac{1}{1 + e^{-1}}\right)
= \frac{e^{-x}}{(1 + e^{-x})^2}
$$